Gemfile

```diff
@@ -37,6 +37,7 @@ gem "sprockets-es6"
 gem "turbolinks", "~> 5"
 gem "uglifier", ">= 1.3.0"
 gem "valid_email2"
+gem "mini_racer"
```

Comment:
Can you please place this alphabetically?

Reply:
Oops, forgot to run `rubocop`, I will add it as a pre-commit hook. Also do you prefer new commits or rebased?